backend/game/cards/discard_cards/victory_point_manipulation_cards/drop_from_high_altitude.py

```diff
@@ -0,0 +1,12 @@
+from game.newcards.discard_card import DiscardCard
+from game.newcards.new_card import NewCard
+
+
+class DropFromHighAltitude(DiscardCard):
+    def __init__(self):
+        NewCard.__init__(self, "Drop from High Altitude", 5, "+ 2[Star] and take control of Tokyo if you don't "
+                                                             "already control it", "")
+
+    def immediate_effect(self, player_that_bought_the_card, other_players):
+        player_that_bought_the_card.update_victory_points_by(2)
+        player_that_bought_the_card.move_to_tokyo()
```

Comment:
Where is the moving the player who was in Tokyo out or Tokyo?

Reply:
Missed it but adding it now

Reply:
I think the order should still be reversed, move everyone out of Toyoko and then move into tokyo

Reply:
I agree with Gerardo. But more importantly, the logic for moving players in and out of tokyo should not live inside this card class. This will be reused elsewhere.

Reply:
Order reversed in recent commit

Reply:
Again, this logic should not live here.